wakuv2/nwaku.go

```diff
@@ -837,42 +851,44 @@ func (w *Waku) retryDnsDiscoveryWithBackoff(ctx context.Context, addr string, su
 	}
 }
 
+*/
+
 func (w *Waku) discoverAndConnectPeers() {
-	fnApply := func(d dnsdisc.DiscoveredNode, wg *sync.WaitGroup) {
-		defer wg.Done()
-		if len(d.PeerInfo.Addrs) != 0 {
-			go w.connect(d.PeerInfo, d.ENR, wps.DNSDiscovery)
-		}
-	}
+	var addrsToConnect []multiaddr.Multiaddr
+	nameserver := "1.1.1.1"
```

Comment:
There should be a nameserver in `w.cfg.Nameserver`. If it does not have a value, use `8.8.8.8` which is what's prefered in mobile.